### DISCUSSION TITLE
Prior information notice used as a call for competition (notice-subtype 10) status flow

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#835] [Q&A] [answered]
Can Prior information notice used as a call for competition (notice-subtype 10) progress to a Contract award notice (notice-subtype 29) and subsequently to a Contract modification notice (notice-subtype 38)? Or since it is a prior information notice, shouldn't it go to contract award?

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Hi,
Since this is a  Prior information notice used as a call for competition, the following step is about the publication of the award notice and your assumption is correct (Notice Subtype 10 => Notice Subtype 29, occasionally followed by Notice Subtype 38).
KR
